### ISSUE TITLE
Add Docker image

### ISSUE THREAD

**Issue by InputObject2** [danielbrendel/hortusfox-web#57]
In order to make this project easier to self-host and develop, consider adding a docker setup and publishing official working images.

**Comment by danielbrendel** [comment]
Wow, nice! I'll review this as soon as I find the time!

**Comment by danielbrendel** [comment]
I just merged the PR into a separate branch. I'll adjust the env settings in the docker-compose file to reflect a standard installation. Then it will go into the main branch.

**Comment by danielbrendel** [comment]
What I observed is there are some strange things:

- When I launch the containers via docker-compose then I get a "connection refused" error from mariadb. This is resolved when I stop the app container and then restart it again. (1)
- The APP_WORKSPACE env var seems to be ignored. When I browse to the web server of the container then it shows that the variable does not contain a valid string but rather NULL (which is reflected in a PHP deprecated warning)

(1)
```shell
Uncaught PDOException: SQLSTATE[HY000] [2002] Connection refused in /var/www/html/vendor/danielbrendel/asatru-php-framework/src/database.php:1168
```

I added a .dockerignore and added the .env file because this might interfere with the docker container in unwanted ways.
